front-end-side/src/Components/ExpensesCard.css

```diff
@@ -0,0 +1,2 @@
+body {
+}
```

Comment:
You could delete this file since it's empty or move card css from data.css

Reply:
Ay ay captain